azure-iot-hub-devicesdk/tests/aio/test_async_clients.py

```diff
@@ -120,6 +142,13 @@ class TestModuleClient(ClientSharedTests):
     def client(self, transport):
         return ModuleClient(transport)
 
+    async def test_instantiation_sets_on_input_message_recieved_handler_in_transport(self, client):
```

Comment:
>recieve [](start = 55, length = 7)

same typo here

Reply:
I hate this spelling so much

---
In reply to: [268366634](https://github.com/Azure/azure-iot-sdk-python-preview/pull/69#discussion_r268366634) [](ancestors = 268366634)